PVReaders is built with cmake and relies on ParaView which does
find_package(Python3...) and thus needs the same cmake options we use
for Python packages.

This relies on the openfoam easyblock setting this.

Åke Sandgren, 2026-10-29
diff -ru OpenFOAM-12-version-12.orig/applications/utilities/postProcessing/graphics/PVReaders/Allwmake OpenFOAM-12-version-12/applications/utilities/postProcessing/graphics/PVReaders/Allwmake
--- OpenFOAM-12-version-12.orig/applications/utilities/postProcessing/graphics/PVReaders/Allwmake	2024-07-06 17:02:47.000000000 +0200
+++ OpenFOAM-12-version-12/applications/utilities/postProcessing/graphics/PVReaders/Allwmake	2026-10-29 08:38:18.456657440 +0100
@@ -24,7 +24,7 @@
         (
             mkdir -p Make/$WM_OPTIONS > /dev/null 2>&1
             cd Make/$WM_OPTIONS
-            cmake ../..
+            cmake $EBOFCMAKEOPTS ../..
             make -j $WM_NCOMPPROCS
         )
     else
@@ -32,14 +32,14 @@
             cd PVblockMeshReader
             mkdir -p Make/$WM_OPTIONS > /dev/null 2>&1
             cd Make/$WM_OPTIONS
-            cmake ../..
+            cmake $EBOFCMAKEOPTS ../..
             make -j $WM_NCOMPPROCS
         )
         (
             cd PVFoamReader
             mkdir -p Make/$WM_OPTIONS > /dev/null 2>&1
             cd Make/$WM_OPTIONS
-            cmake ../..
+            cmake $EBOFCMAKEOPTS ../..
             make -j $WM_NCOMPPROCS
         )
     fi
